html/lib/xaraya/tableddl/xml2ddl-mysql.xsl

```diff
@@ -121,6 +121,12 @@
       <xsl:when test="long">
         <xsl:text>LONGTEXT</xsl:text>
       </xsl:when>
+      <xsl:when test="medium">
+        <xsl:text>mediumtext</xsl:text>
+      </xsl:when>
+      <xsl:when test="blob">
```

Comment:
Make this "binary", as in the code below, and lets keep the data types in capital letters